### DISCUSSION TITLE
Get the most out of your pkgdown website

### DISCUSSION THREAD

**Discussion by Bisaloo** [epiverse-trace/epiverse-trace.github.io#223] [Blog post ideas]
- Add a reference index
- Add the link in `_pkgdown.yml` and `DESCRIPTION` to enable cross-linking
- Use the "Get started" special vignette
- Use bs5 template
- use `fct()` rather than `fct` to enable auto-linking
- don't add custom fields to benefit from automatic translation

This could maybe be cross-posted on R-hub blog.

**Comment by jamesmbaazam** [comment]
Thanks, Hugo. Could expand on the last two bullets?

**Reply by Bisaloo** [reply]
- When writing the vignettes, function should use the markup `function()` (with the parenthesis!) rather than `function`, so a link is automatically added by downlit to the relevant function documentation page. You can see some examples here: https://epiverse-trace.github.io/linelist/articles/linelist.html#tagging-system

- pkgdown provides translation for all the text of its interface by default. This is how we can get ["Primeros Pasos" in sivirep's website](https://epiverse-trace.github.io/sivirep/) rather than "Getting Started", just by changing the language setting in `_pkgdown.yml`, without the need to provide the translation ourselves. For this reason, we should refrain from adding custom text or fields in `_pkgdown.yml` unless there is a very good reason to do so. For each new custom field, we are going to have to provide our own translation.

Does this make sense?
